config/healthcheck.php

```diff
@@ -1,40 +1,46 @@
 <?php
 
 return [
-    /**
+    /*
      * Base path for the health check endpoints, by default will use /
      */
     'base-path' => '',
 
-    /**
+    /*
      * List of health checks to run when determining the health
      * of the service
      */
     'checks' => [
         UKFast\HealthCheck\Checks\LogHealthCheck::class,
         UKFast\HealthCheck\Checks\DatabaseHealthCheck::class,
-        UKFast\HealthCheck\Checks\EnvHealthCheck::class
+        UKFast\HealthCheck\Checks\EnvHealthCheck::class,
     ],
 
-    /**
+    /*
      * A list of middleware to run on the health-check route
      * It's recommended that you have a middleware that only
      * allows admin consumers to see the endpoint.
-     * 
+     *
      * See UKFast\HealthCheck\BasicAuth for a one-size-fits all
      * solution
      */
     'middleware' => [],
 
-    /**
+    /*
      * Used by the basic auth middleware
      */
     'auth' => [
         'user' => env('HEALTH_CHECK_USER'),
         'password' => env('HEALTH_CHECK_PASSWORD'),
     ],
 
-    /**
+    /*
+     * Routename for the healthcheck
+     */
+
+    'routename' => 'healthcheck',
```

Comment:
I'd suggest adding a hyphen between route and name to match the rest of the config.

This will obviously require some changes to tests etc. 

```suggestion
    'route-name' => 'healthcheck',
```